docs/src/library_logical_plan.rs

```diff
@@ -76,3 +79,32 @@ fn plan_builder_1() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
```

Comment:
This file is confusing -- i don't think it is used anymore --- I added https://github.com/apache/datafusion/pull/12081 to remove it